### PR TITLE
remove the GlyphName defined in fea-rs, and use the one in fontdrasil

### PR DESCRIPTION
**Pull request by qxliu76**
for #757

### CODE REVIEW

**Review by rsheeter — APPROVED** [review]

File: fea-rs/src/common/glyph_map.rs [hunk 1]
```diff
@@ -158,9 +161,15 @@ mod sealed {
         }
     }
 
+    impl AsGlyphIdent for SmolStr {
+        fn named(&self) -> Option<&str> {
+            Some(self.as_str())
+        }
+    }
+
```
Comment (cmyr) [inline fea-rs/src/common/glyph_map.rs:169]: I'm not sure why this is necessary?

**Review by cmyr — COMMENTED** [review]
one Q inline, overall looks good!

File: fea-rs/src/common/glyph_map.rs (hunk 1 above)
Comment (cmyr) [reply fea-rs/src/common/glyph_map.rs:169]: ok I see what breaks when I remove it, LGTM :)

**Comment by qxliu76** [comment]
Fixes #757

**Pull request merged**